tests/run-prow-e2e.sh

```diff
@@ -62,5 +62,5 @@ export CYPRESS_MANAGED_OCP_PASS=$(echo $MANAGED_CREDS | jq -r '.password')
 
 export CYPRESS_PROW="true"
 
-echo "Functional Tests"
+echo "Smoke Tests"
```

Comment:
Can we echo Cypress Tests (CYPRESS_TEST_MODE=$CYPRESS_TEST_MODE) here?

Reply:
Actually, it might be better if the mode was displayed as part of `start-cypress-tests.sh` so that from both Prow and Canary smoke/functional we will see the mode in the output.

Reply:
start-cypress-test does have it: https://github.com/open-cluster-management/application-ui/blob/main/tests/start-cypress-tests.sh#L44